x-pack/plugins/ml/server/routes/schemas/anomaly_detectors_schema.ts

```diff
@@ -211,3 +211,10 @@ export const forceQuerySchema = schema.object({
   /** force close */
   force: schema.maybe(schema.boolean()),
 });
+
+export const jobForCloningSchema = schema.object({
+  /** retainCreatedBy boolean */
```

Comment:
Nit - `Whether to retain the created_by custom setting.`

Reply:
Updated in cef6705ed43d877b3e49221073f9fac672ab389c